core/src/main/java/org/eclipse/hono/util/CredentialsObject.java

```diff
@@ -294,6 +294,7 @@ public void checkSecrets(final BiConsumer<String, JsonObject> secretValidator) {
                         checkHashedPassword(secret);
                         secretValidator.accept(getType(), secret);
                     });
+                    break;
```

Comment:
I actually think we should add a checkstyle rule to catch things like this.

Reply:
@ctron are you adding the rule?